src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -126,8 +126,8 @@ protected function get_v1_single_event_html() {
 		$setting = $this->get_template_setting();
 
 		// A number of TEC, and third-party, functions, depend on this. Let's fire it.
-		global $post;
-		do_action( 'the_post', $post );
+		global $post, $wp_query;
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 8ae00244c8b5730f6396c6663917f2c9</sub>